src/main/resources/standardTestCases/exit.test

```diff
@@ -0,0 +1,13 @@
+name: exit
+testFormat: workflowsuccess
+backends: [Jes, Local]
```

Comment:
Hmm I think I want this to go against both actually - is there another way to ask for that ?